stac_fastapi/pgstac/app.py

```diff
@@ -43,22 +43,28 @@
 from stac_fastapi.pgstac.types.search import PgstacSearch
 
 settings = Settings()
-extensions_map = {
+
+# transaction extensions
+trans_extensions_map = {
```

Comment:
do you envision this containing extensions that aren't for transactions? if so it might be more clear to call it `application_extensions_map` (to go along with `application_extensions` further down). or you could rename `application_extensions` to `transaction_extensions`.